pipeline/build.rs

```diff
@@ -8,21 +9,28 @@ use walkdir::WalkDir;
 fn main() {
     build_book_tests("asm");
     build_book_tests("pil");
-    build_reparse_test("asm", "asm");
-    build_reparse_test("pil", "pil");
-    build_reparse_test("asm", "std");
+    build_reparse_test("asm", "asm", Default::default());
+    build_reparse_test("pil", "pil", Default::default());
+    let exclude = if std::env::var("CI").is_ok() {
```

Comment:
I saw another test that does it, but I really don't like this solutions. This introduces more undocumented switched when trying to run the tests.

Can you confirm if increasing the stack size will solve the problem?

Reply:
running in pr-tests solves the problem. How do you propose to increase the stack size?

Reply:
I'm also fine with flipping it around, using some env variable like `POWDR_REPARSE_EXCLUDE_LIST`

Reply:
An alternative solution could be a wrapper to run tests with big stack requirements. Something like:
```
fn run_in_large_stack<F: FnOnce() -> R + Send, R: Send>(f: F) -> R {
    thread::scope(|s| {
    let handle = thread::Builder::new()
        .stack_size(16 * 1024 * 1024) // Set the stack size to 16MB
        .spawn_scoped(s, f)
        .unwrap();
    handle.join().unwrap()
}
```

Reply:
nah, I want to fix it eventually.